addons/mission/functions/fnc_reinforcements.sqf

```diff
@@ -21,9 +21,12 @@
  * [Test_Group_1, false, 0, true, 2000] call MFUNC(reinforcements)
  */
 
-params ["_group", "_state", ["_distance", 0], ["_moveToPlayer", false], ["_searchDistance", 1000]];
+params [["_group", grpNull], "_state", ["_distance", 0], ["_moveToPlayer", false], ["_searchDistance", 1000]];
 
 if (!isServer) exitWith {};
+if (isNull _group) exitWith {
+    WARNING("One of the groups provided does not exist.");
```

Comment:
```suggestion
    WARNING("Group does not exist.");
```
It's not "one of" if function only takes one. Not sure what you could log to identify it though.

Reply:
I tried a few things, couldn't find anything that worked then decided on that.